test_framework/terraform/aws/oracle/main.tf

```diff
@@ -85,6 +85,14 @@ resource "aws_security_group" "lh_aws_secgrp_controlplane" {
     cidr_blocks = ["0.0.0.0/0"]
   }
 
+  ingress {
+    description = "Allow longhorn-ui nodeport"
+    from_port   = 30000
+    to_port     = 30000
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
```

Comment:
Move the codes out of the commit `ci: enable both uio_pci_generic and vfio_pci` and create a separate commit for the change.

Reply:
Moved `vfio` related code to https://github.com/longhorn/longhorn-tests/pull/2056. Thank you!